### ISSUE TITLE
Use shortNames for custom resource

### ISSUE THREAD

**Issue by michalxo** [artemiscloud/activemq-artemis-operator#392]
It might be cool and easier to use shortNames in all ActiveMQArtemis Custom Resource/s
Example
```yaml
names:
    plural: crontabs
    singular: crontab
    kind: CronTab
    shortNames:
    - ct
```

And use it like 
`kubectl get ct -n namespace-name`

```yaml
  group: broker.amq.io
  names:
    kind: ActiveMQArtemis
    listKind: ActiveMQArtemisList
    plural: activemqartemises
    singular: activemqartemis
    shortNames:
    - aa
 ///
    - aaa (address)
    - aas (security)
    - aasd (for scaledown resource)
```

For more information see [shortNames link](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/) .

**Comment by michalxo** [comment]
Maybe having following might be better?
`ac` - ArtemisCloud
`aca` - ACAddress
`acs` - ACSecurity
`acd` - ACScaleDown

**Comment by brusdev** [comment]
I repeated the test using CRDs with different groups. In this case, Kubernetes accepts CRDs with conflicting shortnames. This means CRD full names can be used also if shortnames conflict but conflicting shortnames are only enabled for the first CRD created.

**Comment by brusdev** [comment]
Included in 1.0.12
